index.html

```diff
@@ -1,13 +1,16 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Color Compass - Chingu Voyage 49 Team 16</title>
-    <script defer type="module" src="/src/main.jsx"></script>
+    <title>Vite + React</title>
+    <script src="https://cdn.jsdelivr.net/npm/@jaames/iro@5"></script>
```

Comment:
I am not sure if we have to include `iro` here since we are importing it into our React application I believe it is safe to remove.

Reply:
I trust you!